### PR TITLE
Add whitelist for wogaa

### PR DESCRIPTION
**Pull request by alexanderleegs**

### CODE REVIEW

File: overrides/netlify.toml [hunk 1]
```diff
@@ -73,6 +75,7 @@
         https://unpkg.com
         https://script.wiz.gov.sg/widget.css
         https://script-staging.wiz.gov.sg/widget.css
+        https://*.dcube.cloud
```
Comment (harishv7) [inline overrides/netlify.toml:78]: let's add the exact whitelist as given here for all: https://docs.wogaa.sg/web-implementation/setup-guide/information-service-implementation-test/#step-3-content-security-policies-for-staging

Comment (alexanderleegs) [reply overrides/netlify.toml]: thanks for catching! https://github.com/opengovsg/isomer-build/pull/48/commits/c3a58654690232c0fd0d959e25a664896d9fc77a

File: overrides/netlify.toml [hunk 2]
```diff
@@ -73,6 +75,7 @@
         https://unpkg.com
         https://script.wiz.gov.sg/widget.css
         https://script-staging.wiz.gov.sg/widget.css
+        https://assets.dcube.cloud
```
Comment (harishv7) [inline overrides/netlify.toml:78]: do we need a `/` at the back?

File: overrides/netlify.toml [hunk 3]
```diff
@@ -21,6 +21,7 @@
     Content-Security-Policy = """
       default-src 
         'self'
+        https://*.dcube.cloud
```
Comment (harishv7) [inline overrides/netlify.toml:24]: needs `/` at back?

File: overrides/netlify.toml [hunk 4]
```diff
@@ -143,5 +146,6 @@
         https://staging.ask.gov.sg
         wss://*.zendesk.com
         wss://*.zopim.com
+        https://*.dcube.cloud
```
Comment (harishv7) [inline overrides/netlify.toml:149]: should we add `font-src` above too?

File: overrides/netlify.toml (hunk 3 above)
Comment (alexanderleegs) [reply overrides/netlify.toml]: Fixed in https://github.com/opengovsg/isomer-build/pull/48/commits/09c642abd46fef479de56b26cfef474bfbd67d32!

File: overrides/netlify.toml (hunk 4 above)
Comment (alexanderleegs) [reply overrides/netlify.toml]: I think there's no need for `font-src` - we already have a universal whitelist for that

File: overrides/netlify.toml [hunk 5]
```diff
@@ -55,6 +56,7 @@
         https://script-staging.wiz.gov.sg/widget.js
         wss://*.zendesk.com
         wss://*.zopim.com
+        https://*.dcube.cloud/
```
Comment (harishv7) [inline overrides/netlify.toml:59]: this didn't have the `/` in the instructions https://docs.wogaa.sg/web-implementation/setup-guide/information-service-implementation-test/#step-3-content-security-policies-for-staging

File: overrides/netlify.toml [hunk 6]
```diff
@@ -143,5 +146,6 @@
         https://staging.ask.gov.sg
         wss://*.zendesk.com
         wss://*.zopim.com
+        https://*.dcube.cloud/
```
Comment (harishv7) [inline overrides/netlify.toml:149]: didn't have `/` in the instructions

File: overrides/netlify.toml (hunk 4 above)
Comment (harishv7) [reply overrides/netlify.toml]: okay got it

**Review by harishv7 — APPROVED** [review]
lgtm, with 2 minor comments. Feel free to resolve if it does not apply.

File: overrides/netlify.toml (hunk 5 above)
Comment (alexanderleegs) [reply overrides/netlify.toml:59]: Going to try with the `/` first! if it doesn't work i'll switch it back

**Pull request merged**